index.js

```diff
@@ -28,5 +28,6 @@ export {
   Cancel,
   isAxiosError,
   spread,
-  toFormData
+  toFormData,
+  axios
```

Comment:
Actually, this should not be necessary you can already do this:

`import { axios } from 'axios';`

Reply:
Please see my updated PR description. You can check out my repro at https://github.com/akphi/issue-repo/pull/12
There are 2 problems:
1. For ESM `import`, I don't think `import { axios } from 'axios';` work
2. For TS, we need to update the typings to have the named export - I just updated the PR with this change